output/transform/postgres_relations.go

```diff
@@ -30,6 +30,18 @@ func transformPostgresRelations(s snapshot.FullSnapshot, newState state.Persiste
 			hasParentRelation = true
 		}
 
+		var partStrat snapshot.RelationInformation_PartitionStrategy
+		switch relation.PartitionedBy {
```

Comment:
I think this should be `relation.PartitionStrategy`

Reply:
Good catch